Src/WitsmlExplorer.Frontend/components/QueryEditorWidgetUtils.tsx

```diff
@@ -0,0 +1,113 @@
+import { Icon } from "@equinor/eds-core-react";
+import { createRoot } from "react-dom/client";
+import styled from "styled-components";
+import { getTag } from "./QueryEditorUtils";
+
+/**
+ * Handles the click event for opening a tag in the QueryEditor.
+ * Replaces the self-closing tag with an opening and closing tag and moves the cursor between the tags.
+ * @param editor The Ace editor instance.
+ * @param row The row number of the tag to open.
+ */
+const onClickOpenTag = (editor: any, row: number) => {
```

Comment:
Fix `any` type. `editor: Ace.Editor`

Reply:
We had a discussion and decided to skip this task because it was complicated due to issues with the package.